framework/lib/test_generation/bin/_tool.source

```diff
@@ -10,6 +10,8 @@
 #   separate words.
 #
 
+# shellcheck shell=bash
```

Comment:
I would personally use a shebang line at the top of the file instead.  It is more informative for people and for other tools.